firefox_ui_harness/runners/base.py

```diff
@@ -41,9 +49,43 @@
     'focusmanager.testmode': True,
     'geo.provider.testing': True,
     'javascript.options.showInConsole': True,
+    'marionette.logging': False,
     'security.notification_enable_delay': 0,
     'signon.rememberSignons': False,
     'startup.homepage_welcome_url': 'about:blank',
     'toolkit.startup.max_resumed_crashes': -1,
     'toolkit.telemetry.enabled': False,
 }
+
+
+class FirefoxUITestRunner(BaseMarionetteTestRunner):
+    def __init__(self, **kwargs):
+        BaseMarionetteTestRunner.__init__(self, **kwargs)
+
+        if not self.server_root:
+            self.server_root = firefox_ui_tests.resources
+
+        # Bug 1146847: This needs a refactoring given that our default
+        # preferences are not coming from the command line
+        self.prefs.update(DEFAULT_PREFS)
+
+        if not kwargs.get('e10s'):
+            self.prefs.update({'browser.tabs.remote': False})
```

Comment:
Ups, sorry. That was a bad copy&paste from somewhere else. Thanks for noticing that. I should have indeed used `browser.tabs.remote.autostart` here. Without `--e10s` we don't run multip-process tabs, and with it specified we do. So I hope you are ok with it as pointed out on the bug. Follow-up commit will be up soon.